recharges/admin.py

```diff
@@ -1,5 +1,5 @@
 from django.contrib import admin
 
-from .models import DummyModel
+from recharges.models import RechargeModel
 
-admin.site.register(DummyModel)
\ No newline at end of file
+admin.site.register(RechargeModel)
```

Comment:
Notice that red thingy at the end of the line? It's complaining that there isn't a blank line at the end of the file.  The easiest way to make this go away is to add the following line to your Sublime Text User settings:
`"ensure_newline_at_eof_on_save": true,`